mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/layout/DrawScope.kt

```diff
@@ -40,13 +43,22 @@ public interface DrawScope {
 		background: Color? = null,
 		style: TextStyle? = null,
 	)
+
+	public fun drawText(
+		row: Int,
+		column: Int,
+		string: AnnotatedString,
+		foreground: Color? = null,
+		background: Color? = null,
+		style: TextStyle? = null,
```

Comment:
Do we need these? Should we get rid of them on both overloads?

Reply:
If I understood correctly, then the question is about default values that we really don't use and probably they can be removed

Reply:
I mean the arguments altogether. They seem like a redundant codepath now that you can style within the annotated string. Moreover, they create a precedence problem. If you specify foreground here and as a span which wins?

However, when I look at Compose UI, it seems like they have _three_ colors. You can specify it on a span, on the text as an argument, or on the text within a text style. Ugh. I guess we leave it as-is for now.